docs/encrypted-innodb-tablespace-backups.md

```diff
@@ -1,432 +1,391 @@
-}
+```bash
+xtrabackup --copy-back --target-dir=/data/backup --datadir=/data/mysql \
+--component-keyring-config=/etc/vault.cnf
 ```
 
-The configuration settings can be in either a global or a local configuration file.
+Or using `--move-back`:
+
+```bash
+xtrabackup --move-back --target-dir=/data/backup --datadir=/data/mysql
+xtrabackup --prepare --target-dir=/data/mysql --component-keyring-config=/etc/vault.cnf
+```
 
-??? example "Example of a configuration file in JSON format"
+### Use the keyring file component
 
-     ```json
-     {
-      "timeout": 15,
-      "vault_url": "https://vault.public.com:8202",
-      "secret_mount_point": "secret",
-      "secret_mount_point_version": "AUTO",
-      "token": "58a20c08-8001-fd5f-5192-7498a48eaf20",
-      "vault_ca": "/data/keyring_vault_confs/vault_ca.crt"
-     }
-     ```
+The `keyring_file` component belongs to the component-based MySQL infrastructure. The MySQL server uses this component for encryption. XtraBackup uses the component's configuration file during prepare and restore operations.
 
-Find more information on configuring the `keyring_vault` component in [Use the keyring vault component].
+!!! warning
+   
+    xtrabackup does not copy the keyring file into the backup directory. Copy the keyring file manually before preparing.
 
-The component has no special requirements for backing up a database that
-contains encrypted InnoDB tablespaces.
+#### Phase 1: Backup
 
-The following command creates a backup in the `/data/backup` directory:
+No keyring configuration is needed:
 
-```{.bash data-prompt="$"}
-$ xtrabackup --backup --target-dir=/data/backup --user=root
+```bash
+xtrabackup --backup --target-dir=/data/backup --user=root
 ```
 
-After xtrabackup completes the action, the following message confirms the action:
+#### Phase 2: Prepare
 
-??? example "Confirmation message"
+Provide the keyring configuration file using `--component-keyring-config`. To specify the keyring file data path directly if the configuration file is not in the default location:
 
-    ```{.text .no-copy}
-    xtrabackup: Transaction log of lsn (5696709) to (5696718) was copied.
-    160401 10:25:51 completed OK!
-    ```
-
-### Prepare the backup with the keyring vault component
+```bash
+xtrabackup --prepare --target-dir=/data/backup \
+--component-keyring-config=/var/lib/mysql-keyring/component_keyring_file.cnf
+```
 
-To prepare the backup, the xtrabackup binary must access the keyring. The xtrabackup binary does not communicate with the MySQL server or read the
-default `my.cnf` configuration file. Specify the keyring settings in the command line:
+Alternatively, to specify the keyring data file path directly (not the config file):
 
-```{.bash data-prompt="$"}
-$ xtrabackup --prepare --target-dir=/data/backup --component-keyring-config==/etc/vault.cnf
+```bash
+xtrabackup --prepare --target-dir=/data/backup \
+--component-keyring-config=/var/lib/mysql-keyring/keyring
```

Comment:
/var/lib/mysql-keyring/keyring is wrong. we should use ocmponent_keyring_file.cnf